root/static/scripts/tests/i18n/expand2.js

```diff
@@ -46,6 +46,7 @@ test('expand2', function (t) {
   );
   expandText('An {apple_fruit}', null, 'An {apple_fruit}');
   expandText('An {apple_fruit}', {apple_fruit: 'apple'}, 'An apple');
+  expandText('An {apple fruit}', {'apple fruit': 'apple'}, 'An apple');
```

Comment:
It'd be nice to have two more small tests for `condSubstStart` and `linkSubstStart`. :)

Also, I think `MusicBrainz::Server::Translation::expand` already supports this, but perhaps add a small test for the Perl side, too? There are some existing ones in t/lib/t/MusicBrainz/Server/Translation.pm.

Reply:
Do I understand correctly that `condSubstStart` would be used in tests such as `expandText('{a:%{b:%|}%|}', {a: '0', b: ''}, '00');`? And is `linkSubstStart` tested in tests like this?

```
  expandHtml(
    'A {apple_fruit|<strong>{prefix} APPLE!</strong>}',
    {apple_fruit: 'http://www.apple.com', prefix: 'dang'},
    'A <a href="http://www.apple.com"><strong>dang APPLE!</strong></a>',
  );
```

Reply:
I added a couple tests to the Perl file :)